rollup-bridge-contracts/package.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Can you make this proejct an npm workspace?

Reply:
If you do plz remove prettier and use biome 1.9.4 from the root of the workspaces

Reply:
I guess we can switch to biome in the next PR, together with enabling CI test.
Now compile and deploy scripts required for correct running SC pipeline on nil_dev